modules/@apostrophecms/i18n/i18n/sk.json

```diff
@@ -99,6 +99,9 @@
   "editType": "Upraviť {{ type }}",
   "editor": "Editor",
   "editRelationshipFor": "Upraviť vzťah pre {{ title }}",
+  "editImageAdjustments": "Úpravy obrazu",
```

Comment:
Are these translations coming from google or did you translate the text? We should not add google translations to modules because it makes it more difficult for the volunteer community to identify gaps in what has been reviewed by a human translator.